Unfortunately, the sanitizer functionality depends on a variety of GNU
extensions and headers that simply don't exist on a musl based system. Fixing
this would be a fairly complicated task, though some inspiration could be taken
from dpkg's static 'libcompat' approach of supplying equivalent functionality
that might be missing. A task for another day perhaps.

diff --git a/lib/Makefile.mk b/lib/Makefile.mk
index ed9690d..aacb2b1 100644
--- a/lib/Makefile.mk
+++ b/lib/Makefile.mk
@@ -7,16 +7,16 @@
 #
 #===------------------------------------------------------------------------===#

 SubDirs :=

 # Add submodules.
-SubDirs += asan
+# SubDirs += asan
 SubDirs += builtins
-SubDirs += dfsan
+# SubDirs += dfsan
 SubDirs += interception
-SubDirs += lsan
-SubDirs += msan
+# SubDirs += lsan
+# SubDirs += msan
 SubDirs += profile
-SubDirs += sanitizer_common
-SubDirs += tsan
-SubDirs += ubsan
+# SubDirs += sanitizer_common
+# SubDirs += tsan
+# SubDirs += ubsan
